google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractReadContext.java

```diff
@@ -374,9 +383,15 @@ void initTransaction() {
                 ErrorCode.INTERNAL, "Bad value in transaction.read_timestamp metadata field", e);
           }
           transactionId = transaction.getId();
+          OpenTelemetryTraceUtil.addEvent(
+              openTelemetrySpan,
+              "Transaction Creation Done",
+              OpenTelemetryTraceUtil.getTransactionAnnotations(transaction));
           span.addAnnotation(
               "Transaction Creation Done", TraceUtil.getTransactionAnnotations(transaction));
         } catch (SpannerException e) {
+          OpenTelemetryTraceUtil.addEventWithException(
+              openTelemetrySpan, "Transaction Creation Failed", e);
```

Comment:
Same here also for these: Would it be possible to create some util methods for this, so we can just call one method, and then let the util method call both OT and OC?

Reply:
Done